lib/inspec/profile.rb

```diff
@@ -316,6 +316,25 @@ def check # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metri
       m_unsupported.each { |u| warn.call(meta_path, 0, 0, nil, "doesn't support: #{u}") }
       @logger.info 'Metadata OK.' if m_errors.empty? && m_unsupported.empty?
 
+      # verify that a lockfile is present if we have dependencies
+      if !metadata.dependencies.empty?
+        error.call(meta_path, 0, 0, nil, 'Profile need to be vendored with `inspec vendor`.') if !lockfile_exists?
```

Comment:
Grammar error here:

'Your profile needs to be vendored with `inspec vendor`.'

(need vs needs, and missing article).

Also impacts the functional test.